.github/labeler.yml

```diff
@@ -0,0 +1,29 @@
+documentation:
+  - changed-files:
+      - any-glob-to-any-file: ['docs/**', 'mkdocs.yml']
+docker:
+  - changed-files:
+      - any-glob-to-any-file: ['**/Dockerfile', 'docker-compose.yml']
+element:
+  - changed-files:
+      - any-glob-to-any-file: ['apps/prairielearn/elements/**']
+SQL:
+  - changed-files:
+      - any-glob-to-any-file: ['**/*.sql']
```

Comment:
This seems borderline useless; I'd say the majority of PRs will contain SQL changes. We've only tagged three PRs with this in the entire history of the repository. Maybe we just remove the label?

Reply:
Sure!

Reply:
Resolved in bd2528cfb7d746b36712e68c913a43e29fb9e7df.